srcpkgs/ncurses/template

```diff
@@ -130,3 +151,23 @@ ncurses-term_package() {
 		vmove usr/share/terminfo
 	}
 }
+
+ncurses-libtinfo-libs_package() {
+	shlib_provides=" libtinfo.so.6"
```

Comment:
This should be redundant with the addition to `common/shlibs`, and if it is not - please try it - the leading blank can go away.

Reply:
Oh yeah, you also said that in another comment, i forgot. 
But how do I test if it is redundant?

Reply:
Well, linking against `libtinfo.so.6` from some package which has `ncurses-libtinfo-devel` in its `makedepends` should make it depend on `ncurses-libtinfo-libs`. I think it's ok just to remove that line.